README.md

```diff
@@ -64,6 +64,7 @@ The following query (or config) options are supported:
  - *ignorePartials*: Prevents partial references from being fetched and bundled. Useful for manually loading partials at runtime.
  - *ignoreHelpers*: Prevents helper references from being fetched and bundled. Useful for manually loading helpers at runtime.
  - *precompileOptions*: Options passed to handlebars precompile. See the Handlebars.js [documentation](http://handlebarsjs.com/reference.html#base-precompile) for more information.
+ - *processSource*: Runs the handlebars templates through an html minifier to remove whitespace and reduce the size of the compiled templates.
```

Comment:
`processSource` is a very generic name that makes it hard to guess how exactly the source is being processed. Maybe something like `minifyOutput` would be better?

Reply:
I think `minifyOutput` is a much clearer name. Will make the change.